docs/runner-guest-compatibility.yaml

```diff
@@ -0,0 +1,49 @@
+---
+
+# A list of known, recognized environments for runners and guests
+environments:
+  - CentOS 7
+  - CentOS Stream 8
+  - CentOS Stream 9
+  - CentOS Stream 10
+  - Fedora 39
+  - Fedora 40
+  - Fedora 41
+  - Fedora Rawhide
+  - Alpine 3
+  - Debian 12
+  - Ubuntu 22
+
+#
+# Keys and list entries below are treated as regular expressions tested
+# against a "{runner} + {guest}" string.
+#
+
+# A list of patterns matching unsupported runner/guest combinations.
+unsupported: []
+
+# A list of patterns matching runner/guest combinations where the state
+# of support is unknown.
+unknown:
+  # The following should work as runners, with tmt installed from PyPI,
+  # but we do not know for sure.
+  - 'Alpine .*'
+  - 'Ubuntu .*'
+  - 'Debian .*'
+
+notes:
+  - pattern: 'Fedora Rawhide \+ (CentOS 7|Centos Stream 8)'
+    title: Fedora Rawhide and Ansible 2.17+
+    text: >
+      :ref:`prepare/ansible</plugins/prepare/ansible>`,
+      :ref:`prepare/feature</plugins/prepare/feature>`
+      and :ref:`finish/ansible</plugins/finish/ansible>` will not work
+      out of the box. See their documentation for details and possible
+      workarounds.
+
+  - pattern: '.*? \+ (Alpine|Debian|Ubuntu)'
+    title: Beakerlib not being packaged for non-RHEL OS
```

Comment:
> non-RHEL OS  

non Fedora-based / rpm-based / ...?

Perhaps it would be more accurate to mention that beakerlib framework requires beakerlib being installed or being available in enabled repos?

Reply:
Seems to be addressed by 4080e1d805085338d01adbc4dcb93c7c38aa189a.